wicket/zone-etc/ssh/sshd_config

```diff
@@ -30,7 +30,11 @@ AcceptEnv LANG LC_* OXIDE_PREF_*
 
 AllowUsers wicket support
 
-# Allow "wicket" to log in without a password, and constrain it to the captive shell
+# Allow the use of the SFTP transport protocol
```

Comment:
I think it might help to explain why as the what is somewhat straightforward given the command line. Even if not in the config file, perhaps mentioning it in the description of the issue at least might be useful.

Reply:
I've added more to the configuration file and the PR description.